server/channels/store/sqlstore/team_store.go

```diff
@@ -1687,3 +1687,19 @@ func (s SqlTeamStore) GroupSyncedTeamCount() (int64, error) {
 
 	return count, nil
 }
+
+func (s SqlTeamStore) IsUserAdminOfTeam(userEmail string) (bool, error) {
+	builder := s.getQueryBuilder().Select("COUNT(*)").From("Teams").Where(sq.Eq{"Email": userEmail})
+	query, args, err := builder.ToSql()
+	if err != nil {
+		return false, errors.Wrap(err, "team_tosql")
+	}
```

Comment:
This request should've been checked by the permissions, and here the team e-mail is not reliable source. I would rather rely on permissions system.